fio/src/fioapi/fio.c

```diff
@@ -81,6 +81,7 @@ fio_register
 (
 )
 {
+	system("modprobe fiodriver");
```

Comment:
This can get called multiple times if multiple apps register with the api. Is that ok?

Reply:
I saw a suggestion to use "/sbin/modprobe" in a thread about starting modules from user space programs, as a more secure option.